ui/scripts/build-stack-backend-docker.sh

```diff
@@ -1,7 +1,27 @@
 #!/bin/bash
 
+if  [[ $(cat ~/.docker/config.json  | jq '.auths["ghcr.io"].auth') == 'null' ]]; then
+  echo "In order to run this script and push a new container to the github registry you need to create a personal access token and use it to login to ghcr with docker"
+  echo ""
+  echo "  echo \$MY_PAT | docker login ghcr.io -u USERNAME --password-stdin"
+  echo ""
+  echo "For more information see https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry"
+  echo ""
+  echo "Create a personal access token and log into docker using the above link then try running this script again.s"
+  exit 1
+fi
 
-# this is temporarily being published on rudis docker hub account - @ryarfley
-# TODO: investigate publishing on sifchain repository (possibly GH repo)
-cd .. && docker build -f ./ui/scripts/stack.Dockerfile -t ryardley/sifdevstack01:experimental1 .
-docker push ryardley/sifdevstack01:experimental1
\ No newline at end of file
+echo "Github Registry Login found."
+echo "Building new container..."
+
+NOW=$(date +%s)
```

Comment:
After we have been talking about how to split up this monorepo I was thinking we should make this a git commit hash? That way we can pull down backing stacks for whatever commits we want to test against and have a list of definitive commits we have been tested.